ouroboros-consensus-cardano/src/unstable-cardano-testlib/Test/Consensus/Cardano/ProtocolInfo.hs

```diff
@@ -107,48 +108,45 @@ selectEra Alonzo  HardForkSpec { alonzoHardForkSpec  } = alonzoHardForkSpec
 selectEra Babbage HardForkSpec { babbageHardForkSpec } = babbageHardForkSpec
 selectEra Conway  HardForkSpec { conwayHardForkSpec  } = conwayHardForkSpec
 
-hfSpecProtVer :: Era -> HardForkSpec -> SL.ProtVer
-hfSpecProtVer era = fst . selectEra era
-
-hfSpecTransitionTrigger :: Era -> HardForkSpec -> TriggerHardFork
-hfSpecTransitionTrigger era = snd . selectEra era
-
 stayInByron :: HardForkSpec
 stayInByron =
     HardForkSpec {
-      shelleyHardForkSpec = (SL.ProtVer versionZero 0, TriggerHardForkNotDuringThisExecution)
-    , allegraHardForkSpec = (SL.ProtVer versionZero 0, TriggerHardForkNotDuringThisExecution)
-    , maryHardForkSpec    = (SL.ProtVer versionZero 0, TriggerHardForkNotDuringThisExecution)
-    , alonzoHardForkSpec  = (SL.ProtVer versionZero 0, TriggerHardForkNotDuringThisExecution)
-    , babbageHardForkSpec = (SL.ProtVer versionZero 0, TriggerHardForkNotDuringThisExecution)
-    , conwayHardForkSpec  = (SL.ProtVer versionZero 0, TriggerHardForkNotDuringThisExecution)
+      shelleyHardForkSpec = TriggerHardForkNotDuringThisExecution
+    , allegraHardForkSpec = TriggerHardForkNotDuringThisExecution
+    , maryHardForkSpec    = TriggerHardForkNotDuringThisExecution
+    , alonzoHardForkSpec  = TriggerHardForkNotDuringThisExecution
+    , babbageHardForkSpec = TriggerHardForkNotDuringThisExecution
+    , conwayHardForkSpec  = TriggerHardForkNotDuringThisExecution
     }
 
-versionZero :: SL.Version
-versionZero = SL.natVersion @0
+protocolVersionZero :: SL.ProtVer
+protocolVersionZero = SL.ProtVer versionZero 0
+  where
+    versionZero :: SL.Version
+    versionZero = SL.natVersion @0
 
 -- TODO: could be simplified once we implement something like
 -- ouroboros-network#4115.
```

Comment:
I think this TODO is done?

Reply:
I'm not sure. I wanted to discuss this with @jorisdral, but leave it there until then. I'd be in favour of creating a new issue if we identify concrete simplifications.